docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -56,3 +56,24 @@ PUT /_cluster/settings
 }
 -------------------------------
 // CONSOLE
+
+[[cluster-warning-headers]]
+==== Warning Headers
```

Comment:
I think that the documentation for these settings should be moved to the HTTP docs.